docker-compose.yml

```diff
@@ -22,11 +24,29 @@ services:
       MYSQL_PASSWORD: example
       MYSQL_RANDOM_ROOT_PASSWORD: 'yes'
     volumes:
-      - ./mountPoint/mariadb:/var/lib/mysql
-      - ./provision/mariadb:/docker-entrypoint-initdb.d
-      - ./mountPoint/backup:/backup
+      - "./provision/mariadb:/docker-entrypoint-initdb.d"
+      - "mysql-data:/var/lib/mysql"
+      - "mysql-backups-data:/backup"
       - ./mountPoint/MediaWiki_Backup:/MediaWiki_Backup
-
+  databasebackup:
+    container_name: micro-pkc-databasebackup
+    image: selim13/automysqlbackup:2.6-9
+    restart: unless-stopped # healthcheck on port 18080
+    networks:
+      - common
+    volumes:
+      - "mysql-backups-data:/backup"
+    environment:
+      USERNAME: wikiuser
+      PASSWORD: "example"
+      DBHOST: database
+      DBEXCLUDE: "performance_schema information_schema"
+      CRON_SCHEDULE: "0 22 * * 1-7"
```

Comment:
It would be helpful to have a human-readable/understandable format of this cron schedule.